hummingbot/strategy/pure_market_making/moving_price_band.py

```diff
@@ -0,0 +1,48 @@
+from dataclasses import dataclass
+from decimal import Decimal
+import time
+
+
+@dataclass
+class MovingPriceBand:
+    '''
+    move price floor and ceiling to percentage of current price
+    at every price_band_refresh_time
+    '''
+    enabled: bool = False
+    price_floor_pct: Decimal = -1
+    price_ceiling_pct: Decimal = 1
+    price_band_refresh_time: float = 86400
+    _price_floor: Decimal = 0
+    _price_ceiling: Decimal = 0
+    _set_time: float = 0
+
+    @property
+    def current_timestamp(self) -> float:
+        '''get current timestamp'''
+        return time.time()
```

Comment:
I think the MovingPriceBand logic should use the timestamp from the strategy/connector instead of the system time. All time data used should be related to the tick processing and not related to the real time.

Reply:
OK I have updated to take from strategy.